### ISSUE TITLE
Restart files not moved properly for OpenIFS

### ISSUE THREAD

**Issue by joakimkjellsson** [esm-tools/esm_tools#30]
Hi all

I've added restart file information to oifs.yaml, and it all looks good. But when the "tidy_and_resubmit" process starts, ESM-Tools has no idea what the restart files are. I get a bunch of errors like:
"File is not in list: ...."

If you look at "restart_out_files" as it is written in the log file (attached), it looks fine the first time. But ESM-Tools lists all the variables several times, and each time it seems to lose some files. 
So in my .log file, I've got this on line 1713: 
```
restart_out_files:
  BLS: BLS
  LAW: LAW
  rcf: rcf
  srf: srf
```

Then, on line 2760 it lists the same stuff again, but this time without the "srf" files. 
```
restart_out_files:
  BLS: BLS
  LAW: LAW
  rcf: rcf
```

Then on line 6705 I have a lot of lines like this:
```
File is not in list: /work/bb0519/b350090/esm-experiments//oifstest3/run_20080101-20080105/work/srf000000040000.0111
```

Eventually the job crashes with 
```
File: /work/bb0519/b350090/esm-experiments//oifstest3/run_20080101-20080105/input/oifs/199_4/o3chem_l91/O3CHEM08_V2.9
Traceback (most recent call last):
  File "/pf/b/b350090/.local/bin/esm_runscripts", line 8, in <module>
    sys.exit(main())
  File "/pf/b/b350090/.local/lib/python3.6/site-packages/esm_runscripts/cli.py", line 168, in main
    Setup()
  File "/pf/b/b350090/.local/lib/python3.6/site-packages/esm_runscripts/esm_sim_objects.py", line 54, in __call__
    self.tidy(*args, **kwargs)
  File "/pf/b/b350090/.local/lib/python3.6/site-packages/esm_runscripts/esm_sim_objects.py", line 715, in tidy
    self.copy_all_results_to_exp()
  File "/pf/b/b350090/.local/lib/python3.6/site-packages/esm_runscripts/esm_sim_objects.py", line 778, in copy_all_results_to_exp
    os.mkdir(destination_path)
FileNotFoundError: [Errno 2] No such file or directory: '/work/bb0519/b350090/esm-experiments//oifstest3/input/oifs/199_4/o3chem_l91'
```

I'm attaching all the files for my "script" dir, i.e. the yaml and log files etc. 

Does anyone know why the "srf" files are not recognised and why restart files don't get moved? 
I can see that the file "rcf" ends up in the restart/oifs/ directory where it should, so that's fine. But the "srf" and "BLS" and "LAW" files should be there too. 
Is the problem with the globbing? 

Many thanks in advance for any help!
Cheers
Joakim 

PS. I've added "txt" as file ending for all attached files. Github would not accept them otherwise...

[oifstest3_compute_20080101-20080105.sad.txt](https://github.com/esm-tools/esm_tools/files/4567492/oifstest3_compute_20080101-20080105.sad.txt)

[hostfile_srun.txt](https://github.com/esm-tools/esm_tools/files/4567488/hostfile_srun.txt)
[monitoring_file.out.txt](https://github.com/esm-tools/esm_tools/files/4567489/monitoring_file.out.txt)
[oifs-43r3-mistral-initial-daily.yaml.txt](https://github.com/esm-tools/esm_tools/files/4567490/oifs-43r3-mistral-initial-daily.yaml.txt)
[oifstest3_compute_20080101-20080105_20919483.log.txt](https://github.com/esm-tools/esm_tools/files/4567491/oifstest3_compute_20080101-20080105_20919483.log.txt)

**Comment by JanStreffing** [comment]
Hey Joakim, nice progress. In your runscript the restart unit is set to one month, while the total runtime is only 5 days. Can you check if setting the restart unit to days makes a difference?

Cheers, Jan

**Comment by joakimkjellsson** [comment]
Hi Jan. I will check, but as I understand the "restart_rate" variable does not do anything for OpenIFS. The total run length is set by "initial_data" and "final_date" and the "nday" variable sets the restart rate. 
I do get restart files in my work directory and all the names look ok. But somehow ESM-Tools "loses" the filenames and then has no idea what the "srf" files are. 

Cheers 
Joakim

**Comment by seb-wahl** [comment]
I close this issue as saving restarts from OIFS is fixed in ESM-Tools 4.2
